src/mainwindow.cpp

```diff
@@ -3700,7 +3714,7 @@ void MainWindow::setUseNativeWindowFrame(bool useNativeWindowFrame)
 void MainWindow::setHideToTray(bool enabled)
 {
     m_hideToTray = enabled;
-    m_settingsDatabase->setValue(QStringLiteral("hideToTrayEnabled"), enabled);
+    m_settingsDatabase->setValue(QStringLiteral("hideToTray"), enabled);
```

Comment:
I'd put this into a different commit, as it's a change unrelated to the problem.

Reply:
Ok, I made 2 commits fixing these problems.